test/modules/layers/test_quantisation.py

```diff
@@ -63,8 +68,12 @@ def test_quantised_output(self):
                 ],
             ]
         )
+        expected_flat = expected.permute(0, 2, 1).contiguous().view(-1, 5)
+        expected_codebook = torch.Tensor([2, 2, 0, 2, 1, 3]).type(torch.LongTensor)
 
         assert_expected(actual, expected)
+        assert_expected(actual_flat, expected_flat)
```

Comment:
is this redundant based on the previous check?

Reply:
normally yes but wanted to make sure that `get_quantised` returns the correct flattened quantised vectors